src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.csproj

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="Humanizer" />
     <PackageReference Include="MediatR" />
     <PackageReference Include="Microsoft.AspNetCore.JsonPatch" />
-    <PackageReference Include="SFA.DAS.VacancyServices.Wage" />
+    <PackageReference Include="SFA.DAS.VacancyServices.Wage" Version="1.0.35" />
```

Comment:
https://github.com/SkillsFundingAgency/das-apim-endpoints/blob/master/Directory.Packages.props this file needs updating with the version, the project files all use this

Reply:
Corrected in commit 603b0b61b.